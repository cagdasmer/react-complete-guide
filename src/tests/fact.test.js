import React from "react";

import { shallow } from "enzyme";
import Fact from "../components/fact";

test("Should correctly render the Fact component", () => {
  const text = "This is a fact";
  let wrapper = shallow(<Fact text={text} />);
  expect(wrapper).toMatchSnapshot();
});
