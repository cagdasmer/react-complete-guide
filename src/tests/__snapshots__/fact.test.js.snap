// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly render the Fact component 1`] = `
<div>
  This is a fact
</div>
`;
